### ISSUE TITLE
Input forms are not valid anymore

### ISSUE THREAD

**Issue by akroii** [contao/contao#774]
**Affected version(s)**
4.8
**Description**  
For w3c Input forms (input type="text") are not valid anymore
```
<form method="post" enctype="application/x-www-form-urlencoded" class="projektanfrage">
  <div class="formbody">
  <input type="hidden" name="FORM_SUBMIT" value="auto_form_3">
  <input type="hidden" name="REQUEST_TOKEN" value="BmppLsDxv2pYj9oioa8y0qNdTFg-PDAlW23Ptxv1T1A">
  <input type="hidden" name="Referrer" value="https://www.newface-werbeagentur.de">
  <input type="hidden" name="IP" value="234234234234234">
  <input type="hidden" name="Abgesendet_am" value="19.09.2019 12:48:08">
  <div class="widget widget-text">
  <label for="ctrl_12">
  Ihr Name </label>
  <input type="text" name="name" id="ctrl_12" class="text" value="" max="0" min="0">
  </div>
  <div class="widget widget-text">
  <label for="ctrl_13">
  Ihre Telefonnummer </label>
  <input type="text" name="phone" id="ctrl_13" class="text" value="" max="0" min="0">
  </div>
  <div class="widget widget-text mandatory">
  <label for="ctrl_14" class="mandatory">
  <span class="invisible">Pflichtfeld </span>Ihre Email-Adresse<span class="mandatory">*</span>
  </label>
  <input type="email" name="email" id="ctrl_14" class="text mandatory" value="" required max="0" min="0">
  </div>
  <div class="widget widget-textarea">
  <label for="ctrl_15">
  Ihre Anfrage </label>
  <textarea name="message" id="ctrl_15" class="textarea" rows="4" cols="40"></textarea>
  </div>
  <div class="widget widget-checkbox small mandatory">
  <fieldset id="ctrl_199" class="checkbox_container small mandatory">
  <input type="hidden" name="verarbeitung_akzeptiert" value="">
  <span><input type="checkbox" name="verarbeitung_akzeptiert" id="opt_199_0" class="checkbox" value="verarbeitung_akzeptiert" required> <label id="lbl_199_0" for="opt_199_0">Ich habe die <a href="datenschutz" title="Datenschutzerklärung">Datenschutzerklärung</a> zur Kenntnis genommen. Ich stimme zu, dass meine Angaben und Daten zur Beantwortung meiner Anfrage elektronisch erhoben, gespeichert und verarbeitet werden. Die Daten werden nach abgeschlossener Bearbeitung Ihrer Anfrage gelöscht. Hinweis: Sie können Ihre Einwilligung jederzeit für die Zukunft per E-Mail an ccccc widerrufen.</label></span>
  </fieldset>
  </div>
  <div class="submit_container">
  <div class="widget widget-submit">
  <button type="submit" id="ctrl_16" class="submit">Absenden</button>
  </div>
  </div> </div>
  </form>
```
W3c validator:
![Bildschirmfoto 2019-09-19 um 12 50 32](https://user-images.githubusercontent.com/8830861/65238011-14e93980-dadc-11e9-8bad-a3074d833a6c.png)

Settings in Contao Backend:
![Bildschirmfoto 2019-09-19 um 12 53 11](https://user-images.githubusercontent.com/8830861/65238168-74474980-dadc-11e9-9b9e-424c3020c5a8.png)

**How to reproduce**  
1. Go to contao.demo.org
2. Create form
3. Check frontend code

or just validate: ([https://demo.contao.org/en/applications.html](https://demo.contao.org/en/applications.html))

**Comment by aschempp** [comment]
I guess the problem is that `min` and `max` are always added even if the input type does not match.

**Comment by leofeyer** [comment]
Fixed in 16475eed748430c60bf0d9a6ba90203672f02776.
